frontend/app/scripts/check-external-links.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('node:fs');
+const axios = require('axios');
+const { externalLinks } = require('./../src/data/external-links.js');
+
+const processDynamicUrl = url =>
+  // Format dynamic URL and replace with valid value.
+  url.replace('$version', '1.31.1');
```

Comment:
what is the version supposed to do? we could import package.json and get it from there?

Reply:
Ok that would be tricky, if we keep manual we have to update on every release, if we make it automatic it might fail in between tagging and publishing a release.

Reply:
maybe I should get it from github, like how we did in download page? 
<img width="306" alt="image" src="https://github.com/rotki/rotki/assets/26648140/70f85a35-f11f-49a0-983c-e5594bef9f18">

Reply:
I do it this way

> maybe I should get it from github, like how we did in download page? <img alt="image" width="306" src="https://private-user-images.githubusercontent.com/26648140/291946403-70f85a35-f11f-49a0-983c-e5594bef9f18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyMTU4MDcsIm5iZiI6MTcwMzIxNTUwNywicGF0aCI6Ii8yNjY0ODE0MC8yOTE5NDY0MDMtNzBmODVhMzUtZjExZi00OWEwLTk4M2MtZTU1OTRiZWY5ZjE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIyVDAzMjUwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlMmFiNzg4YzE4YzU4ODllOWU2M2Q1MmExOTE2OWRjMzI5MThkMmMyNzI0M2NlYjFjZDg5NjhmNjUxZDZhNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Jf36cgzV5pdYmxFEha8qcRnCp0VeIdi9tWnjxrCqSH4">